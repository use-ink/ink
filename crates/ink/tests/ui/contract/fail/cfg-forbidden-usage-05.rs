#[ink::contract]
mod contract {
    #[ink(storage)]
    pub struct Contract {}

    impl Contract {
        #[ink(constructor)]
        pub fn constructor() -> Self {
            Self {}
        }

        #[ink(message)]
        pub fn message1(&self) {}

        #[ink(message)]
        #[cfg(target_pointer_width = "32")]
        pub fn message2(&self) {}
    }
}

fn main() {}
