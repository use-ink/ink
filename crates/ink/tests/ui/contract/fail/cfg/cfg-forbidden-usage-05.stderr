error: This `cfg` attribute is not allowed.

       Allowed in `#[cfg(â€¦)]`:
       - `test`
       - `feature` (without `std`)
       - `any`
       - `not`
       - `all`
  --> tests/ui/contract/fail/cfg/cfg-forbidden-usage-05.rs:18:15
   |
18 |         #[cfg(target_pointer_width = "32")]
   |               ^^^^^^^^^^^^^^^^^^^^
